src/main.cpp

```diff
@@ -116,6 +123,9 @@ void fill_targets(){
 }
 
 void render(){
+  // macOS Mojave rerender fix
+  int windowNeedsUpdating = 2;
```

Comment:
did you mean for this to be `static`? otherwise this will never actually decrease as `windowNeedsUpdating--` is called only once after its usage in `render()`

Reply:
also, please use a more... descriptive... name than this. `render()` is pretty big.